### DISCUSSION TITLE
(Technical issue)What kind of data should we obtain for fine-tuning?

### DISCUSSION THREAD

**Discussion by BillyHeYifan** [ZIYANGSONG2003/AI-Enhanced-WordPress-Development-Toolkit#165] [General]
Should we only rely on the WordPress code database to generate the code for training?
Or can we use any other channel to generate code for training, like prompting other LLM to generate training data?

**Comment by ZIYANGSONG2003** [comment]
"Great question, @BillyHeYifan! While relying on the WordPress code database is a solid approach for obtaining relevant data, expanding the dataset by using other channels like prompting other LLMs (Large Language Models) can potentially improve the diversity and robustness of the training data. Just be sure that the generated data is of high quality and aligns with the structure and requirements of the fine-tuning task. It would also be useful to document the sources of data used for better transparency and reproducibility."
